package.json

```diff
@@ -142,12 +144,17 @@
   },
   "homepage": "https://threejs.org/",
   "dependencies": {
+    "@codemirror/commands": "^6.1.0",
+    "@codemirror/lang-javascript": "^6.0.2",
+    "@codemirror/lint": "^6.0.0",
+    "@codemirror/state": "^6.1.1",
+    "@codemirror/theme-one-dark": "^6.0.0",
+    "@codemirror/view": "^6.2.1",
     "acorn": "1.0.1",
-    "codemirror": "5.65.2",
-    "esprima": "3.1.3",
-    "jsonlint": "1.6.3",
-    "signals": "1.0.0",
-    "ternjs": "0.22.0"
+    "codemirror": "^6.0.1",
+    "eslint4b-prebuilt": "^6.7.2",
+    "esprima": "^3.1.3",
+    "signals": "1.0.0"
```

Comment:
1. esprima should be there, it seems b/c of https://github.com/Exabyte-io/three.js/pull/13/files#diff-164db2a22a4c894e29fdccb6def3bfcf77c385084b8a0f6bdbce66146d2f5a88R2 - I re-added
2. Please double-check package-lock.json to only contain what's needed and re-commit if necessary

Reply:
Hmm... strange I deleted these packages and then re-check them in the wave app and it worked for me even without esprisma.